tools/pot/openvino/tools/pot/graph/graph_utils.py

```diff
@@ -3,12 +3,13 @@
 
 import os
 import tempfile
+import ngraph as ng
 from copy import deepcopy
 from openvino.tools.mo.graph.graph import Graph
 from openvino.tools.mo.utils.ir_reader.restore_graph import restore_graph_from_ir, save_restored_graph
 from openvino.tools.mo.utils.logger import init_logger
 from openvino.inference_engine import IECore  # pylint: disable=E0611
-from openvino.offline_transformations import ApplyPOTTransformations  # pylint: disable=import-error,no-name-in-module
+from openvino.offline_transformations_pybind import apply_pot_transformations # pylint: disable=import-error,no-name-in-module
```

Comment:
will we rename `offline_transformations_pybind` then?

Reply:
yes, will do it in separate PR

Reply:
ping